GameData/RealismOverhaul/Engine_Configs/Astris_Config.cfg

```diff
@@ -51,7 +51,7 @@
 		}
 		CONFIG
 		{
-			// source: http://www.b14643.de/Spacerockets/Diverse/European_Rocket_engines/engines.htm
```

Comment:
Unless you found this previous source to be bad, I would suggest keeping this source in addition to the new one.